sympy/polys/ring_series.py

```diff
@@ -572,6 +574,111 @@ def _coefficient_t(p, t):
             p1[monomial_div(expv, expv1)] = p[expv]
     return p1
 
+def rs_fast_series_reversion(p,x,n):
```

Comment:
Is there any reason that this function should not replace the existing function? Perhaps the old function could be renamed if there is a reason to keep it around.